kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/TimerIntegrationTest.java

```diff
@@ -91,4 +93,29 @@ public void testTimerStartEvent() throws Exception {
         assertTrue(distance2 < 10000);
 
     }
-}
+
+
+    @Test(timeout = 60 * 1000)
+    public void testTimerRoolbackTimerCancel() throws Exception {
+        String containerId = "timer-project-" + runtimeStrategy;
+        createContainer(containerId, releaseId, new KieServerConfigItem(KieServerConstants.PCFG_RUNTIME_STRATEGY, runtimeStrategy, String.class.getName()));
+
+        this.processClient.startProcess(containerId, "error-handling.test-rollback");
+        List<ProcessInstance> startedInstances = queryClient.findProcessInstancesByContainerId(containerId, null, 0, 10, null, false);
+
+        assertEquals(1, startedInstances.size());
+
+        // this should fail
+        try {
+            List<TaskSummary> summary = this.taskClient.findTasks(USER_JOHN, null, null);
+            TaskSummary taskSummary = summary.get(0);
```

Comment:
Task is not assigned to user "john" in the bpmn, so it retrieves no elements.
I think it's better to that these 2 lines out of the try-catch to avoid non-expected exceptions